codec-http/src/main/java/io/netty/handler/codec/rtsp/RtspHeaders.java

```diff
@@ -24,6 +24,7 @@
  * Standard RTSP header names and values.
  */
 @Deprecated
+@SuppressWarnings("deprecation")
```

Comment:
Could be, I usually supress warnings of use of deprecated other classes in a deprecated class because typically that otherwise results in a lot of warnings that are obsolete and which may hide warnings that are relevant. Your call, I'll revert it if you prefer so.